cmd/mr_list.go

```diff
@@ -74,18 +75,22 @@ func mrList(args []string) ([]*gitlab.MergeRequest, error) {
 		num = -1
 	}
 
+	// gitlab lib still doesn't have search by assignee and author username
+	// for merge requests, because of that we need to get the ID for both.
 	if mrAssignee != "" {
-		_assigneeID, err := lab.UserIDByUserName(mrAssignee)
+		assigneeID, err := lab.UserIDByUserName(mrAssignee)
 		if err != nil {
 			log.Fatal(err)
 		}
-		assigneeID = &_assigneeID
-	} else if mrMine {
-		_assigneeID, err := lab.UserID()
+		mrAssigneeID = &assigneeID
+	}
+
+	if mrAuthor != "" {
```

Comment:
I was thinking about allowing `--author` without argument as a fancy shorthand for "authored-by-me", but it's probably a bad idea given that `--assignee=""` means "not assigned to anybody".

Reply:
yea. I think it would be again confusing.
I thought about creating `--authored-by-me` and `--assigned-to-me` or something like that later.. but at the same time it isn't that important considering both are just shorthand for something possible with `--author` and `--assignee`.